src/utils.jl

```diff
@@ -38,18 +38,29 @@ function atanpos(y, x)
 end
 
 """
-    fixlon(lon)
+    checklat(lat)
+Checks if the latitude is in the range `[-90°,90°]` and returns it if yes or throws an error otherwise.
 
-Fix the longitude to be in the range `[-180°,180°]`.
+!!! note
+The check has a tolerance of `1e-6°` around +90° and -90°, so if `90° < abs(lat) < 90.000001°` the function returns either +90° or -90° depending on the sign of the input.
```

Comment:
Can you please comment on the choice of 1e-6? Your tests with CRS conversion led to this practical number?

Reply:
It was just a random placeholder to make tests pass (tests with error actually had 1e-9 or so error), I think the actual value would be discussed/investigated better